src/operation.rs

```diff
@@ -0,0 +1,28 @@
+use coinbase_mesh::models::{AccountIdentifier, Amount, Currency, Operation, OperationIdentifier};
+use convert_case::{Case, Casing};
+
+use crate::{OperationStatus, OperationType, TransactionStatus};
+
+pub fn operation(
```

Comment:
In other parts of the code, I see quite a bit of `clone`ing of arguments supplied to this factory fn. Should we instead update the parameters to be references? This way we could get rid of some of the boilerplate elsewhere.